src/utils/url.utils.ts

```diff
@@ -104,3 +111,57 @@ export const getWebsocketURL = () => {
 
     return `wss://${serverURL}/websockets/v3?app_id=${getAppId()}&l=${language}&brand=${AppIDConstants.appBrand}`;
 };
+
+/**
+ * Extracts query parameters from the URL by parsing the current window's URL search parameters for the specified key.
+ * It returns the query parameters associated with the given key.
+ *
+ * @returns {string | null} A string containing query parameter associated with the given key.
+ */
+export const getURLParameters = (key: queryParameters) => {
```

Comment:
```suggestion
export const getURLParameters = (key: queryParameters[]) => {
```
Probably be more convenient allow user to pass an array of keys to do multiple key checks at once.
Then returning an array of key value pair:

```
[{ key: "lang", value: "es" }, { key: "action", value: null }]
```

Reply:
Valid point @yashim-deriv 🙏

Reply:
Base on our discussion let's keep it like this for now later if anyone need to pass multiple keys to this function then we can apply the changes.